pages/home.md

```diff
@@ -42,22 +42,18 @@ permalink: /
             </figure>
           </div>
           <div class="column is-offset-1 is-5">
-            <h2 class="title">All of your data, under your control</h2>
+            <h2 class="title">Pods: All your data, under your control</h2>
```

Comment:
Wanted to immediately bring up Pod - since it's central to all the blurbs in these sections.

Reply:
```suggestion
            <h2 class="title">Your pod: All your data, under your control</h2>
```